woocommerce/woocommerce-gutenberg-products-block
# Storybook and TS migration of `PriceSlider` component (#5253)

This PR adds stories for the `PriceSlider` component and migrates it to TypeScript. It also migrates a bunch of associated functions and components.

Fixes #5253

### Testing

### Manual Testing

How to test the changes in this Pull Request:

1. Run `npm run storybook`
2. Explore the [PriceSlider stories](http://localhost:6006/?path=/story/woocommerce-blocks-base-components-priceslider--default). You should see one story: it should be fully interactive, showing all the controls with the appropriate types and defaults.

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).
